doc/content_item_fields.md

```diff
@@ -16,7 +16,8 @@ from the content store API can be found in
 
 ## `base_path`
 
-A string. Present in all contexts.
+A string. Present in the request URL in the publishing context, stored as `_id`
```

Comment:
i think you want to say: "Present in the request URL in the storing context, ..."
there are only 3 contexts we've established: storing, notifying, and retrieving.